src/Configuration/Dotenv.hs

```diff
@@ -156,3 +158,6 @@ lookUpDuplicates (x:xs) =
   if x `elem` xs
     then forbidDuplicates x
     else lookUpDuplicates xs
+
+orderList :: [(String, String)] -> [(String, String)]
+orderList = toList . fromList
```

Comment:
I'm curious... why not just use `sortOn fst`?
ref: https://hackage.haskell.org/package/base-4.18.0.0/docs/Data-List.html#v:sortOn

Reply:
Oh I see It's because you want to remove duplicates... but [line 90](https://github.com/stackbuilders/dotenv-hs/pull/161/files#diff-28752b99ee8c901d2f3dead57a3943b5fe22dc7c15bf7ec272eee5e646f4843fR91) tells me that duplicates may be allowed :thinking: I think you're introducing an undesired side effect here... or what am I missing?

Reply:
Well, the idea here is to pick the last set value in the list if it contains duplicates. For that I make use of the `Data.Map` to drop duplicates picking the last value for a repeated key but it has the side effect that also orders the list. It seems that when duplicates were allowed it was using the first value and ignoring the last one. Behavior that this PR aims to change.

I see your point about the name, maybe it can be named  something like `mapVars`?

Reply:
Hey @anddriex I would suggest a different name. Something like `nubByLastVar` since it's removing duplicates, and taking the last variable. WDYT?

Reply:
Sounds good. Thanks for your suggest!